wrapanapi/systems/rhevm.py

```diff
@@ -581,7 +581,7 @@ def wait_for_ok_status(self, timeout=1800):
             message="template is OK",
             delay=10)
 
-    def deploy(self, vm_name, cluster, timeout=900, power_on=True, **kwargs):
+    def deploy(self, vm_name, cluster, timeout=900, power_on=True, initialization=None, **kwargs):
```

Comment:
Can you add a small docblock about the `initialization`?

Reply:
done

Reply:
Thanks!